background.js

```diff
@@ -0,0 +1,11 @@
+let thisBrowser = (window.browser) ? browser: chrome;
```

Comment:
In a Firefox extension window.browser and window.chrome are both undefined. The global object isn't actually `window` inside of a WebExtension. Thus, this expression will always result in `chrome`